libraries/aws.rb

```diff
@@ -93,7 +93,10 @@ def specific_machine_options(component, _count = nil)
         return [] unless @node['delivery-cluster'][component]
         options = []
         options << { bootstrap_options: { instance_type: @node['delivery-cluster'][component]['flavor'] } } if @node['delivery-cluster'][component]['flavor']
+        options << { bootstrap_options: { security_group_ids: @node['delivery-cluster'][component]['security_group_ids'] } } if @node['delivery-cluster'][component]['security_group_ids']
+        options << { image_id: @node['delivery-cluster'][component]['image_id'] } if @node['delivery-cluster'][component]['image_id']
         # Specify more specific machine_options to add
+        options
```

Comment:
It is required because of the trailing conditional (and I think it makes it easier to read anyway)
```
irb(main):001:0> foo = []
=> []
irb(main):002:0> foo << 1
=> [1]
irb(main):003:0> foo << nil
=> [1, nil]
irb(main):004:0> foo
=> [1, nil]
irb(main):005:0> foo << 12 if false
=> nil
```